Java/.gitignore

```diff
@@ -1,3 +1,3 @@
 .idea
-.iml
+*.iml
```

Comment:
why it's necessary?

Reply:
Change is not related to task. Just for my convenience. Without this, file aircompany.iml was marked as untracked file